assets/javascripts/discourse/components/ai-bot-header-panel.js

```diff
@@ -0,0 +1,45 @@
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Component from "@ember/component";
+import { ComposeAiBotMessage } from "discourse/plugins/discourse-ai/discourse/lib/ai-bot-helper";
+
+import I18n from "I18n";
+
+export default class AiBotHeaderPanel extends Component {
+  @service siteSettings;
+  @service composer;
+  @service appEvents;
+
+  @action
+  async composeMessageWithTargetBot(target) {
+    this._composeAiBotMessage(target);
+  }
+
+  @action
+  async singleComposeAiBotMessage() {
+    this._composeAiBotMessage(
+      this.siteSettings.ai_bot_enabled_chat_bots.split("|")[0]
```

Comment:
is there a possibility `this.siteSettings.ai_bot_enabled_chat_bots` would be an empty string?

Reply:
Also it could probably use `this.enabledBotOptions` you defined later

Reply:
would not be rendered, but yeah we should pass that in to the panel ... this specific code though was dead code... removed it